controller/api/apis.go

```diff
@@ -2597,28 +2597,43 @@ type RESTWorkloadInterceptData struct {
 	Intercept *RESTWorkloadIntercept `json:"intercept"`
 }
 
-type TagDetail struct {
-	ID          string `yaml:"id" json:"id"`
-	Title       string `yaml:"title" json:"title"`
-	Description string `yaml:"description" json:"description"`
+type RESTAvaiableComplianceFilter struct {
+	AvailableFilter []string `json:"available_filter"`
 }
 
 type RESTBenchCheck struct {
-	TestNum     string                   `json:"test_number"`
-	Category    string                   `json:"category"`
-	Type        string                   `json:"type"`
-	Profile     string                   `json:"profile"`
-	Scored      bool                     `json:"scored"`
-	Automated   bool                     `json:"automated"`
-	Description string                   `json:"description"`
-	Remediation string                   `json:"remediation"`
-	Tags        []map[string][]TagDetail `json:"tags,omitempty"`
+	TestNum     string                      `json:"test_number"`
+	Category    string                      `json:"category"`
+	Type        string                      `json:"type"`
+	Profile     string                      `json:"profile"`
+	Scored      bool                        `json:"scored"`
+	Automated   bool                        `json:"automated"`
+	Description string                      `json:"description"`
+	Remediation string                      `json:"remediation"`
+	Tags        []string                    `json:"tags,omitempty"`
```

Comment:
The reason TagsV2 is added is to provide backward compatibility.
But the change of Tags' type breaks backward compatibility again. Why?

Reply:
The Tag in 5.3.3 is add is the type I change here, please take a look at this [PR](https://github.com/neuvector/neuvector/pull/1289)